scrape/server/flask_app/controllers/users.py

```diff
@@ -1 +1,10 @@
-from flask_app import app
\ No newline at end of file
+from flask_app import app
+from scrape.server.flask_app.repo import User as UserModel
```

Comment:
Rename this to UserRepo since it's imported from `scrape.server.flask_app.repo` and not `scrape.server.flask_app.model`

Reply:
Hey Ray, just wondering. I changed line 2 to UserRepo so I also changed line 8 from u = UserModel(config) to u = UserRepo(config) is that right?

Reply:
Yup